internal/image/fixtures/test-ubuntu.Dockerfile

```diff
@@ -0,0 +1 @@
+FROM ubuntu:22.04@sha256:ed1544e454989078f5dec1bfdabd8c5cc9c48e0705d07b678ab6ae3fb61952d2
```

Comment:
Can we pick a slim image to reduce the number of packages?

Reply:
I don't see ubuntu has official slim images, but this one is not big, only 28mb similar size to Debian slim.